db-drda/src/main/java/com/splicemachine/db/impl/drda/DRDAConnThread.java

```diff
@@ -1944,8 +1949,53 @@ private int parseACCSEC() throws  DRDAProtocolException
                         if (securityMechanism != CodePoint.SECMEC_USRIDPWD &&
                                 securityMechanism != CodePoint.SECMEC_USRIDONL)
                         {
+							if (securityMechanism == CodePoint.SECMEC_KERSEC) {
+
+
+								try {
+									user = UserGroupInformation.getCurrentUser();
+
+									Exception exception = user.doAs(new PrivilegedAction<Exception>() {
+										@Override
+										public Exception run() {
+											try {
+												// Get own Kerberos credentials for accepting connection
+												GSSManager manager = GSSManager.getInstance();
+												Oid krb5Mechanism = new Oid("1.2.840.113554.1.2.2");
+												GSSCredential serverCreds = manager.createCredential(null,
+														GSSCredential.DEFAULT_LIFETIME,
+														krb5Mechanism,
+														GSSCredential.ACCEPT_ONLY);
+
+
+											/*
+											 * Create a GSSContext to receive the incoming request
+											 * from the client. Use null for the server credentials
+											 * passed in. This tells the underlying mechanism
+											 * to use whatever credentials it has available that
+											 * can be used to accept this connection.
+											 */
+												gssContext = manager.createContext(
+														(GSSCredential) serverCreds);
+											} catch (Exception e) {
+												return e;
+											}
+											return null;
+										}
+									});
+
+									if (exception != null) {
+										throw exception;
+									}
+								} catch (Exception e) {
+									println2Log(null, session.drdaID, e.getMessage());
```

Comment:
I'm not sure, actually I think it's disabled, I just followed the same pattern as the DRDA code. But we should look into it...